modules/810-documentation/templates/deployment.yaml

```
@@ -49,6 +51,60 @@ spec:
             subPath: nginx.conf
           - name: tmp
             mountPath: /tmp
+          - mountPath: /app/modules/en
+            name: modules-docs-en
+          - mountPath: /app/modules/ru
+            name: modules-docs-ru
+      - name: builder
+        image: {{ include "helm_lib_module_image" (list $ "docsBuilder" "docs-builder") }}
+        command:
+          - "/app/server"
+        env:
+          - name: HUGO_ENVIRONMENT
+            value: production
+          - name: POD_IP
+            valueFrom:
+              fieldRef:
+                fieldPath: status.podIP
+          - name: POD_NAME
+            valueFrom:
+              fieldRef:
+                apiVersion: v1
+                fieldPath: metadata.name
+          - name: POD_NAMESPACE
+            valueFrom:
+              fieldRef:
+                apiVersion: v1
+                fieldPath: metadata.namespace
+          - name: CLUSTER_DOMAIN
+            value: {{ .Values.global.clusterConfiguration.clusterDomain }}
+        ports:
+          - containerPort: 8081
+            name: http
+            protocol: TCP
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8081
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 8081
+        resources:
+          requests:
+            {{- include "helm_lib_module_ephemeral_storage_logs_with_extra" 10 | nindent 12 }}
+  {{- if not ( .Values.global.enabledModules | has "vertical-pod-autoscaler-crd") }}
+            {{- include "builder_resources" . | nindent 12 }}
+  {{- end }}
+        securityContext:
+          allowPrivilegeEscalation: false
```

```suggestion
        securityContext:
          allowPrivilegeEscalation: false
```
This is required to use one of the following helpers https://github.com/deckhouse/lib-helm/tree/main/charts/helm_lib#module-security-context

You can find examples in the other modules in the repo.

---

Fixed